LANCER RPG (1.8)/LANCER.html

```diff
@@ -1682,4 +1700,40 @@
 			</div> <!--frame column 2-->
 		</div> <!--row wrapper-->
 	</div><!--FRAME tab-->
-</div> <!-- main container div<-->
\ No newline at end of file
+</div> <!-- main container div<-->
+
+<rolltemplate class="sheet-rolltemplate-lancer">
+	<div class="sheet-container sheet-color-{{color}}">
+	<div class="sheet-header">
+		{{#name}}<div class="sheet-name">{{name}}</div>{{/name}}
+		{{#tags}}<div class="sheet-tags">{{tags}}</div>{{/tags}}
+	</div>
+	<div class="sheet-content">
+		<div class="sheet-rng-dmg">
+		{{#range}}[ <img class="sheet-icon" src="https://raw.githubusercontent.com/meepletea/lancer/master/icons/range.png" />{{range}} ] {{/range}}
```

Comment:
These URLs need to change to `https://github.com/Roll20/roll20-character-sheets/blob/master/LANCER%20RPG%20(1.8)/images/*`